corehq/apps/commtrack/util.py

```diff
@@ -122,20 +110,26 @@ def _needs_commtrack_config(domain):
                 caption='Stock-out',
             ),
         ],
-    )
-
-    if requisitions_enabled:
-        config.requisition_config = get_default_requisition_config()
+    ).save()
 
-    config.save()
 
-    program = get_or_create_default_program(domain.name)
+def _enable_commtrack_previews(domain):
+    for toggle_class in (
```

Comment:
Shouldn't this be reversible operation? Other question probably is - is it possible for a project to go back-n-forth from CommTrack, without losing setup/data? (Though this is an edge case scenario)